packages/klighd-core/src/bookmarks/bookmark-panel.tsx

```diff
@@ -30,8 +30,10 @@ import { Bookmark, GoToBookmarkAction, CreateBookmarkAction, DeleteBookmarkActio
  */
 @injectable()
 export class BookmarkPanel extends SidebarPanel {
-
-    @inject(TYPES.IActionDispatcher) private actionDispatcher: IActionDispatcher;
+    // sets the bookmarkpanel at the bottom
+    // hirarchy is: first = -10; middle = 0; last = 10;
+    readonly position = 10; // --> last position
```

Comment:
this would not be last, if another panel was added with a bigger number, comment could be more generic here (higher number leads to lower priority or similar)